### PR TITLE
Feature/comments popup

### LINKED ISSUE #14: [4pt]  -Display Comments pop up with selected item's details - Student B
Create a comments popup only with the top section (displaying details of the selected item)

Screenshot 2021-06-27 at 22 10 45

When the popup loads, the webapp retrieves data from the selected API and shows details about the selected item.

Add event to button prepared by your teammate that will open your popup. You need to communicate with your team member to make sure that you do not block each other (most likely the easiest solution is to merge the list of items feature to the dev branch before you try to add that event).

### PR DESCRIPTION
**Pull request by jssol**
Create a comments popup only with the top section (displaying details of the selected item)

Closes #14

## :warning: Very important note :warning:

I have encapsulated previous functionalities into functions, may you check them and incorporate them in your workflow because we need to organize our code well.

Check out the live demo [here](https://jssol.github.io/api-based-webapp)

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -243,6 +254,24 @@ body {
...
+.m-year {
+  background-color: rgba(62, 62, 62, 0.2);
+  border-radius: 5px;
+  padding: 4px;
+  font-size: 14px;
+  font-weight: 600;
+  margin-left: 10px;
+}
+
+/* End popup section */
```
Comment (Ismailco) [inline src/index.css]: same here

File: src/index.css [hunk 2]
```diff
@@ -26,6 +26,17 @@ body {
...
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  z-index: -10;
+  background-image: url(./assets/img/bg.jpg);
+  background-position: top;
+  background-size: cover;
+  filter: blur(5px);
+}
+
```
Comment (Ismailco) [inline src/index.css]: don't change the structure, leave this block of code after the body

**Review by Ismailco — COMMENTED** [review]
Very good work 👍🏼

File: src/index.css (hunk 2 above)
Comment (jssol) [reply src/index.css]: I changed it in the `index.js` file, you should check it out

Comment (Ismailco) [reply src/index.css]: Ok, I forget to pull the changes!

**Review by Ismailco — APPROVED** [review]
Ok, you're doing great work!
You can merge the PR

**Pull request merged**